src/robusta/core/reporting/base.py

```diff
@@ -37,14 +39,27 @@ def from_severity(severity: str) -> "FindingSeverity":
             return FindingSeverity.HIGH
 
         raise Exception(f"Unknown severity {severity}")
-    
+
     def to_emoji(self) -> str:
-        if self == FindingSeverity.DEBUG: return "🔵"
-        elif self == FindingSeverity.INFO: return "🟢"
-        elif self == FindingSeverity.LOW: return "🟡"
-        elif self == FindingSeverity.MEDIUM: return "🟠" 
-        elif self ==  FindingSeverity.HIGH: return "🔴"
+        if self == FindingSeverity.DEBUG:
+            return "🔵"
+        elif self == FindingSeverity.INFO:
+            return "🟢"
+        elif self == FindingSeverity.LOW:
+            return "🟡"
+        elif self == FindingSeverity.MEDIUM:
+            return "🟠"
+        elif self == FindingSeverity.HIGH:
+            return "🔴"
+
 
+class VideoLink(BaseModel):
+    url: str
+    name: str = "See more"
+
+    def to_json(self):
```

Comment:
Pydantic has builtin to_json
When using BaseModel we get to/from json/yaml for free